botbot/apps/plugins/core/logger.py

```diff
@@ -10,6 +12,12 @@ class Config(config.BaseConfig):
         help_text="Don't log lines starting with this string"
     )
 
+def should_ignore_text(text, ignore_prefix):
+    return any(
+        (re.match(prefix, text, flags=re.IGNORECASE) is not None)
```

Comment:
I meant in general - chat messages aren't likely going to be combing in faster than a regular expression can be compiled. And I'm not sure the best way to store precompiled regular expressions in this case, since there doesn't appear to be a function that gets called to reload the plugin when the config changes... So we should leave it like this for now.